.github/workflows/stale-issues-and-prs.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Mark and close stale issues and identify stale pull requests'
+# This workflow was adapted from the uberon stale-issue workflow: https://github.com/obophenotype/uberon/blob/master/.github/workflows/stale-issues.yml
+# See https://github.com/actions/stale for info about the action
+
+on:
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          days-before-issue-stale: 630
+          days-before-issue-close: 30
+          close-issue-label: 'autoclosed-unfixed'
+          stale-issue-message: 'This issue has not seen any activity in the past 2 years. It will automatically 30 days from now if no action is taken.'
+          close-issue-message: 'This issue has been closed automatically because it has not been updated in over two years. Please re-open if you still need this to be addressed.'
+          start-date: '2015-07-02T00:00:00Z'
+          days-before-pr-stale: 90
+          # days-before-pr-close: 30
+          stale-pr-message: 'This PR has not seen any activity in the 90 days and has been marked as stale. If it is no longer needed, please close the PR. Otherwise, please update the PR with a status update.'
```

Comment:
```suggestion
          stale-pr-message: 'This PR has not seen any activity in 90 days and has been marked as stale. If it is no longer needed, please close the PR. Otherwise, please update the PR with a status update.'
```